flink-core/src/main/java/org/apache/flink/api/common/functions/util/AbstractRuntimeUDFContext.java

```diff
@@ -93,6 +93,15 @@ public int getIndexOfThisSubtask() {
 	}
 
 	@Override
+	public String getTaskNameWithSubtasks() {
```

Comment:
Since this method may be called rather often, I would create this string once and return it then.